src/paymentservice/index.js

```diff
@@ -63,9 +57,9 @@ server.addService(health.service, new health.Implementation({
 
 server.addService(hipsterShopPackage.hipstershop.PaymentService.service, { charge: chargeServiceHandler })
 
-server.bindAsync(`0.0.0.0:${process.env['PAYMENT_SERVICE_PORT']}`, grpc.ServerCredentials.createInsecure(), (err, port) => {
+server.bindAsync(`0.0.0.0:${process.env['PAYMENT_SERVICE_PORT'] ?? 50051}`, grpc.ServerCredentials.createInsecure(), (err, port) => {
```

Comment:
I don't think we want to hardcode a default port for any of the services outside of the .env file.
```suggestion
server.bindAsync(`0.0.0.0:${process.env['PAYMENT_SERVICE_PORT']}`, grpc.ServerCredentials.createInsecure(), (err, port) => {
```